package.json

```diff
@@ -151,6 +151,26 @@
         "command": "server.editServer",
         "title": "Edit Server",
         "category": "Servers"
+      },
+      {
+        "command": "server.distribution.run",
+        "title": "Run on Server",
+        "category": "Distribution"
```

Comment:
the naming "Distribution" is confusing to me. Do you mean an application which is bundled as war/ear/jar? To me this is more of an "application artifact(s)", "artifact(s)" or "application".